packages/core/js-client/src/jsPeer/__test__/avm.spec.ts

```diff
@@ -5,19 +5,19 @@ import { handleTimeout } from '../../particle/Particle.js';
 describe('Basic AVM functionality in Fluence Peer tests', () => {
     it('Simple call', async () => {
         await withPeer(async (peer) => {
-            const res = await new Promise<string[]>((resolve, reject) => {
-                const script = `
+            const script = `
                 (call %init_peer_id% ("print" "print") ["1"])
             `;
-                const particle = peer.internals.createNewParticle(script);
-
+            const particle = await peer.internals.createNewParticle(script);
+            
+            const res = await new Promise<string>((resolve, reject) => {
                 if (particle instanceof Error) {
                     return reject(particle.message);
                 }
 
                 registerHandlersHelper(peer, particle, {
                     print: {
-                        print: (args: Array<Array<string>>) => {
+                        print: (args: Array<string>) => {
```

Comment:
Why is it string? It should be `unknown`. Absolutely anything can be passed to `Console.print`.

Reply:
this type is specific for test. test always passes string so coercing this to string is fine.

Reply:
makes sense, thx